js/src/util.js

```diff
@@ -11,12 +11,84 @@ const isCharWhitespace = (charCode) => {
   return whitespaces.includes(charCode)
 }
 
+const isCharNewLine = (charCode) => {
+  return charCode === 10 || charCode === 13
```

Comment:
What if the line break consists of these two charaters `/r/n`? Is this case handled propely?

Reply:
This function only checks the char code of a single character